Lombiq.DataTables/Controllers/TableController.cs

```diff
@@ -17,7 +17,7 @@ public class TableController : Controller
     public TableController(IEnumerable<IDataTableDataProvider> dataTableDataProviders) =>
         _dataTableDataProviders = dataTableDataProviders;
 
-    [Route("/Admin/DataTable/{providerName}/{queryId?}")]
+    [Route("Admin/DataTable/{providerName}/{queryId?}")]
```

Comment:
Instead of a fixed prefix this would be better with `AdminOptions.AdminUrlPrefix`, see the OC examples.

Reply:
That can only be done in `Startup.Configure` instead of the `[Route]`˙attribute, correct?

Reply:
Yep.

Reply:
:'(